sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -84,6 +85,18 @@ internal static bool TryGetRetryIntervalTimespan(Response? httpResponse, out Tim
             return false;
         }
 
+        internal static byte[] GetSerializedContent(IEnumerable<TelemetryItem> body)
+        {
+            using var content = new NDJsonWriter();
+            foreach (var item in body)
+            {
+                content.JsonWriter.WriteObjectValue(item);
+                content.WriteNewLine();
+            }
+
+            return content.ToBytes().ToArray();
```

Comment:
what namespace is the `NDJsonWriter` from?
I'm not familiar with this type.

It's a red flag for me that we're adding new serialization logic here.
What serializer was being used before now?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/NDJsonWriter.cs is a part of our project and safe to use.

Reply:
Not something new: https://github.com/Azure/azure-sdk-for-net/blob/2cfa587f85f734fc7da3573a7349b5f642e73e44/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/ApplicationInsightsRestClient.cs#L83-L96
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/NDJsonWriter.cs

Reply:
Thanks for clarifying. 

nit: since this is nearly identical serialization logic, we should consider moving this to a helper class. 
Not a blocker for this PR.